recipes/jemalloc/all/conanfile.py

```diff
@@ -130,6 +131,9 @@ def _patch_sources(self):
                                   "\t$(INSTALL) -m 755 $(objroot)lib/$(LIBJEMALLOC).$(SOREV) $(BINDIR)\n"
                                   "\t$(INSTALL) -m 644 $(objroot)lib/libjemalloc.a $(LIBDIR)")
 
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
```

Comment:
Patches need to be applied in build.

Reply:
This function (_patch_sources) is being called by build()
https://github.com/TheSalvator/conan-center-index/blob/a773d5ad0a656e29a6fcf37f4bae326ef6897c93/recipes/jemalloc/all/conanfile.py#L138

Reply:
Thanks I missed that in the diff collapse